libmarch/src/python/CMakeLists.txt

```diff
@@ -12,8 +12,8 @@ set(MARCH_PY_SOURCES
     march.cpp
     march_gas.cpp
 )
-pybind11_add_module(march ${MARCH_PY_SOURCES})
-target_link_libraries(march PRIVATE ${SCOTCH_LIBRARIES})
-install(TARGETS march DESTINATION ${SOLVCON_ROOT})
+pybind11_add_module(libmarch ${MARCH_PY_SOURCES})
```

Comment:
Rename the top-level module name to `libmarch`, since `march` could be too generic.